testdata/configtest.json

```diff
@@ -1229,12 +1229,17 @@
     },
     "useGlobalFormat": true,
     "assetTypes": [
-     "spot"
+     "spot",
```

Comment:
Could you also please update `config_example.json` with a valid coinbasepro setup (minus API keys)?